nemo/collections/asr/parts/utils/eval_utils.py

```diff
@@ -15,10 +15,18 @@
 import json
 from typing import Optional, Tuple, Union
 
+from torchmetrics.text import SacreBLEUScore
```

Comment:
These need to be guarded because we don't always install sacrebleu and rouge

Reply:
won't the dependencies be installed when installing torchmetrics? I didn't know where torchmetrics requires sacrebleu and rouge, thought it uses its own implementaion

Reply:
I don't want to take a chance with those two. They are annoying libraries to install on other machines than Linux

Reply:
I searched through torchmetrics dependencies and didn't find sacrebleu or rouge. I also looked into its implementation, the SacreBLEUScore implementation doesn't need other dependencies, while the ROUGEScore implementation only needs NLTK, so I think it's safe to add those two metrics?

Reply:
Nltk has issues on clusters, I'll send you a thread on slack later. Make the class with imports nltk run only inside of the place that's needed, not at the top of the file.

Reply:
Nvm above. Nltk itself is fine.